.ci/build-kit/build_and_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
```

Comment:
I wrote to Andreas about this point and we decided that he will refactor this in the CI refactor for all repos. Because in the other build_and_test shell scripts this is also done in the same way as here. That's why I would leave it as it is here.